diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index b251713..b7e6d96 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -20,14 +20,14 @@ else()
   SET(RUST_LIB "${RUST_DIR}/${RUST_TARGET_DIR}/librust.a")
 endif()
 
-add_custom_command(
-  OUTPUT "${RUST_LIB}"
-  COMMAND ${Cargo_EXECUTABLE} build ${RUST_BUILD_FLAG}
-  DEPENDS QLivePlayer-Lib/src/lib.rs
-          QLivePlayer-Lib/src/implementation.rs
-          QLivePlayer-Lib/src/interface.rs
-  WORKING_DIRECTORY "${RUST_DIR}"
-)
+# add_custom_command(
+#   OUTPUT "${RUST_LIB}"
+#   COMMAND ${Cargo_EXECUTABLE} build ${RUST_BUILD_FLAG}
+#   DEPENDS QLivePlayer-Lib/src/lib.rs
+#           QLivePlayer-Lib/src/implementation.rs
+#           QLivePlayer-Lib/src/interface.rs
+#   WORKING_DIRECTORY "${RUST_DIR}"
+# )
 add_custom_target(rust_target DEPENDS "${RUST_LIB}")
 
 set(qlp_file_SRCS
